test/extended/testdata/service_catalog/ups-deployment.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ups-deployment-template
+objects:
+- apiVersion: extensions/v1beta1
+  kind: Deployment
```

Comment:
extensions/v1beta1 are deprecated for a long time and removed in kube already, followed by removal in 4.4, correct is apps/v1